src/QsCompiler/Transformations/CallGraph.cs

```diff
@@ -136,7 +134,7 @@ internal static bool TryCombineTypeResolutions(out TypeParameterResolutions comb
     /// </summary>
     public class CallGraphEdge
```

Comment:
In reference to `CombineEdges` method.

Reply:
Copied from post by @irinayat-MS: From the summary comment I'd name this method something like `CombinePathIntoSingleEdge`. On the other hand, if for correctness of this method it's required to provide a continuous path of edges in `edges` array, why would you need to filter the type resolutions by target? Aren't all type resolutions on a given edge specific to its target call node?

Reply:
Copied from post by @irinayat-MS: Oh! I think I'm getting it! The goal is to select the subset of type resolutions on the path that actually reach the target. Correct?

Reply:
Right. Many different unrelated resolutions may happen between node `A` and node `Z`. But this method is just supposed to get an abstraction of the relationship between `A` and `Z`, so it needs to filter out all the unrelated resolutions and combine the related resolutions.

Reply:
I like your name suggestion better. It draws attention to the fact that this acts more on a path than random edges.

Reply:
I've renamed the method to `CombinePathIntoSingleEdge`, per your suggestion.